cli/src/main.rs

```diff
@@ -3008,18 +3008,15 @@ mod tests {
         )
         .with_header(HeaderName::ACCEPT, "application/n-triples")?
         .build();
-        server.test_body(
-            request,
-            "<http://example.com/s> <http://example.com/p> \"\u{1f468}\" .\n",
-        )?;
+        server.test_body(request, "<s> <p> \"\u{1f468}\" .\n")?;
```

Comment:
I'm not sure application/n-triples supports BASE IRIs

Reply:
Indeed. But the lenient mode should not fail on relative IRIs and just allow them. Thank you!